project.py

```diff
@@ -267,6 +267,32 @@ def get_ssn_assignment_prefix():
                 ssn_prefix[int(ls[0])] = ls[1]
     return ssn_prefix
 
+def get_ssn_prefix_count(ssn_array: list) -> dict:
+    print(get_ssn_assignment_prefix())
+    #get ssn to state mapping
+    prefix_to_state_dict = get_ssn_assignment_prefix()
+
+    state_dict = get_state_abbrev()
+  
+    #hold each state name to pass to get_state_abbrev_freq()

```

Comment:
A python list can be used in situations like this.  Your implementation works fine, but, could be simplified using lists and the join method like this:
```
    #hold each state name to pass to get_state_abbrev_freq()
    state_names = []

    prefix_count = {}

    for i in range(len(ssn_array)):
      #get prefix of each ssn
      prefix = int(ssn_array[i][0:3])

      state_name = prefix_to_state_dict[prefix].strip()
      if state_name in state_dict:
          state_names.append(state_name)
    state_names_str = ','.join(state_names)
    print("State names: " + state_names_str)
```